docs/LSPApi.md

```diff
@@ -60,6 +60,7 @@ public class MyLanguageServerFactory implements LanguageServerFactory {
 | URI getFileUri(VirtualFile file)                             | Returns the file Uri from the given virtual file and null otherwise (to use default `FileUriSupport.DEFAULT.getFileUri`).                                                                                                                                                                                                                                                                      | `null`            |
 | VirtualFile findFileByUri(String fileUri)                    | Returns the virtual file found by the given file Uri and null otherwise (to use default `FileUriSupport.DEFAULT.findFileByUri`).                                                                                                                                                                                                                                                               | `null`            |
 | boolean isCaseSensitive(PsiFile file)                        | Returns `true` if the language grammar for the given file is case-sensitive and `false` otherwise.                                                                                                                                                                                                                                                                                             | `false`           | 
+| boolean serverReportsOffsetOnlyPositions(PsiFile file)       | Returns `true` if the language server reports `Position` values as pure offsets and `false` otherwise.                                                                                                                                                                                                                                                                                         | `false`           | 

```

Comment:
This is the doc for the new client config option to tell LSP4IJ that this specific server reports `Position` values that are of the form `[0,offset]`.